docs/manual/getting_started.rst

```diff
@@ -1162,6 +1262,220 @@ deleting the root entity of all the others: the domain participant.
+
+CMake uses the CMakeLists.txt in the helloworld directory to create
+makefiles that fit the native platform.
+
+Subsequently, build the example. We recommend you provide the config of
+Visual Studio:
+
+.. code-block:: bash
+
+    cmake --build . --config "Release"
+
+The *DDS CXX Hello World* example application can now be found in the
+``helloworld\build\Release`` directory, use the method in `Test your CXX
+installation <#test-your-cxx-installation-for-native-installation>`__
+to check if the application runs successfully.
+
+**Note:** If the *DDS CXX Hello World* application fails, please check
+the `environment variable <#test-your-cxx-installation-for-native-installation>`__ is set up correctly.
+
+
+Hello World! Code Anatomy in Python
+===================================
+This section details how to create your first DDS application in Python API.
+
+Your first Python DDS application
+''''''''''''''''''''''''''''''''''
+
+Let's enter the world of DDS by making our presence known. We will not worry about configuration or what DDS does under the hood but write a single message. 
+To publish anything to DDS we need to define the type of message first. Suppose you are worried about talking to other applications that are not necessarily running Python. In that case, you will use the CycloneDDS IDL compiler, but for now, we will manually define our message type directly in Python using the ``cyclonedds.idl`` tools:
+
+.. code-block:: python3
+    :linenos:
+
+    from dataclasses import dataclass
+    from cyclonedds.idl import IdlStruct
+
+    @dataclass
+    class Message(IdlStruct):
+        text: str
+
+
+    name = input("What is your name? ")
+    message = Message(text=f"{name} has started his first DDS Python application!")
+
+
+With ``cyclonedds.idl`` write typed classes with the standard library module `dataclasses <python:dataclasses>`. 
+For this simple application, you put in a piece of text, but this system has the same expressive power as the OMG IDL specification, allowing you to use almost any complex datastructure.
+
+To send your message over a DDS domain, carry out the following steps:
+
+1. Join the DDS network using a DomainParticipant
+2. Define which data type and under what name you will publish your message as a Topic
+3. Create the DataWriter that publishes that Topic
+4. And finally, publish the message.
+
+
+.. code-block:: python3
+    :linenos:
+
+    
+    from cyclonedds.topic import Topic
+    from cyclonedds.pub import DataWriter
+
+    participant = DomainParticipant()
+    topic = Topic(participant, "Announcements", Message)
+    writer = DataWriter(participant, topic)
+
+    writer.write(message)
+
+You have now published your first message successfully! However, it is hard to tell if that did anything, since we don't have anything set up that is listening. Let's make a second script that takes messages from DDS and prints them to the terminal:
```

Comment:
```suggestion
You have now published your first message successfully! However, it is hard to tell if that did anything, since we don't have anything set up to listen for incoming messages. Let's make a second script that takes messages from DDS and prints them to the terminal:
```